### PR TITLE
Feature/alert rules v2

### PR DESCRIPTION
**Pull request by QBY-MarkusMaring**
# Description

This PR updates the alert rules to v2

# Change overview (tick true):

- [x] This introduces backward incompatible changes
- [ ] This adds a new backward compatible Feature
- [ ] This fixes a Bug

# Version information: 

- Previous Version: `4.2.1`

- Next Version based on [Semantic Versioning](https://semver.org/#summary) (see above): `5.0.0`

# How Has This Been Tested?

- [x] Apply of no_event_pipeline example was successfull
- [x] Queries are tested

# Checklist:

- [x] I have run tests and documented them above
- [x] I have performed a self-review of my own code
- [ ] I have updated the documentation
- [x] I have updated the CHANGELOG

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -15,25 +15,42 @@ resource "azurerm_monitor_action_group" "eventpipeline" {
...
-    operator  = "GreaterThan"
-    threshold = 0
+
+  scopes                = [var.log_analytics_workspace.id]
+  description           = each.value.description
+  enabled               = true
+  severity              = 0
+  skip_query_validation = true
+
+  evaluation_frequency = "PT5M"
```
Comment (QBY-ThimoLimpert) [inline main.tf:35]: I think it is reasonable to allow to set the frequency, as you can set the duration.

File: queries/windows_filespace.kusto [hunk 2]
```diff
@@ -30,6 +30,6 @@ InsightsMetrics
 | extend script_name = "n/a"
 | extend script_version = "n/a"
 | extend value = round(Val, 2)
-| extend additional_information = strcat("Only ", (value), " % Diskspace free. Managed by ",(tags_managedby_s),".")
+| extend additional_information = strcat("Only ", (value), " % Diskspace free. Managed by ", (tags_managedby_s), ".")
 | extend affected_entity = Mountpoint
-| project TimeGenerated = now(), _ResourceId, state, affected_object, monitor_package, monitor_name, monitor_description, script_name, script_version, threshold, value, affected_entity, additional_information
\ No newline at end of file
+| project _ResourceId, RawData = strcat("TimeGenerated:", now(), ";_ResourceId:", _ResourceId, ";state:", state, ";affected_object:", affected_object, ";monitor_package:", monitor_package, ";monitor_name:", monitor_name, ";monitor_description:", monitor_description, ";script_name:", script_name, ";script_version:", script_version, ";threshold:", threshold, ";value:", value, ";affected_entity:", affected_entity, ";additional_information:", additional_information)
```
Comment (QBY-ThimoLimpert) [inline queries/windows_filespace.kusto:35]: I don't think we should "define" ou own format. Use json instead.

**Review by QBY-ThimoLimpert — CHANGES_REQUESTED** [review]

File: queries/windows_filespace.kusto (hunk 2 above)
Comment (QBY-MarkusMaring) [reply queries/windows_filespace.kusto:35]: Good suggestion. Will not be done in this PR sadly

**Review by QBY-SvenHaas — APPROVED** [review]
All changes have been verified. OK.

**Pull request merged**